dask_expr/collection.py

```diff
@@ -122,29 +122,35 @@ def __getitem__(self, other):
 
     def __dask_graph__(self):
         out = self.expr
-        out = out.simplify()
+        out = out.optimize(fuse=False)
         return out.__dask_graph__()
 
     def __dask_keys__(self):
         out = self.expr
-        out = out.simplify()
+        out = out.optimize(fuse=False)
```

Comment:
**Just a note** (**not** a suggestion for this PR):  We added the `simplify` step to `__dask_graph__`/`__dask_keys__` to deal with the fact that there may be "abstract" nodes in the expression graph. It **may** make sense to only call `lower` here (instead of `optimize`). Where lower would do something like:

```python
    def lower(self):
        expr = self
        while True:
            out = expr.lower_once()
            if out._name == expr._name:
                break
            expr = out
        return expr
```

Such a change would probably require a lot of tests to change, but may make the most sense (not sure about this yet).